docs/bricks_overview.rst

```diff
@@ -66,9 +66,9 @@ Lazy initialization
 In the example above we configured the :class:`.Linear` brick during
 initialization. We specified input and output dimensions, and specified the
 way in which weight matrices should be initialized. But consider the
-following case, which is quite common: We want to take the output of one
+following case, which is quite common: we want to take the output of one
```

Comment:
Haha, @fvisin mentioned this in https://github.com/bartvm/blocks/issues/338 so I will say the same thing. I know that in French it needs to be a small letter, but not in English. It's arguably a slightly ambiguous case, because some style guides (e.g. Chicago) will argue that it should be small letter if there's only a single full sentence following the colon, but in this case the following sentence contains several clauses, so IMHO it really needs to be a capital :)